sqlglot/optimizer/merge_subqueries.py

```diff
@@ -119,6 +119,19 @@ def _mergeable(outer_scope, inner_select, leave_tables_isolated, from_or_join):
     Returns:
         bool: True if can be merged
     """
+
+    def _get_window_expressions_in_where_clause():
+        window_expressions = list(inner_select.find_all(exp.Window))
+        window_alias_names = [window.parent.alias_or_name for window in window_expressions]
```

Comment:
you probably want to make this a set? {window.parent.alias_or_name for w in window}